lib/indexers/tasks/decorators/open-closed.js

```diff
@@ -0,0 +1,8 @@
+const closed = ['resolved', 'rejected', 'discarded-by-asru', 'discarded-by-applicant', 'withdrawn-by-applicant'];
```

Comment:
Considered adding workflow as a dependency here to get the closed statuses but seems overkill?